src/com/orderinchaos/Obstacle.java

```diff
@@ -0,0 +1,65 @@
+package com.orderinchaos;
+
+import java.util.Scanner;
+
+public class Obstacle {
+  // FIELDS/ ATTRIBUTES
+  private boolean pass;
+  private Riddle riddle;
+  private static final int ATTEMPTS = 4;
+
+
+  public Obstacle(Riddle riddle) {
+    setRiddle(riddle);
+  }
+
+  // ON YOUR MARK
+  public boolean isPass() {
+    return pass;
+  }
+
+  public void setPass(boolean pass) {
+    this.pass = pass;
+  }
+
+  public int getAttempts() {
+    return ATTEMPTS;
+  }
+
+  private void setRiddle (Riddle riddle) {
+    this.riddle = riddle;
+  }
+
+  // BUSINESS METHODS
+  public void displayRiddle() {
+    int attemptsLeft = getAttempts();
+    Scanner sc = new Scanner(System.in);
+    System.out.println(riddle.getQuestion());
```

Comment:
Let's modify/overload our util scanner and handle user interaction for test purposes